src/controllers/department.controller.js

```diff
@@ -12,15 +12,16 @@ const { version } = require('chai');
   const addDepartment = async (req ,res ,next) => {
     console.log(req.body);
     try {
-    const validate  = await departmentValidation.validateAsync(req.body);
-    console.log("------------>",validate);
```

Comment:
Never use console.log, 
you can use console.info or console.error instead, if required. Try to eleminate the console usage.